diff -urN aa/gnome-volume-manager-0.8.2/configure.in gnome-volume-manager-0.8.2/configure.in
--- aa/gnome-volume-manager-0.8.2/configure.in	2004-03-21 00:55:45.000000000 +0100
+++ gnome-volume-manager-0.8.2/configure.in	2004-03-21 23:31:32.431000152 +0100
@@ -16,7 +16,7 @@
 fi
 changequote([,])dnl
 
-ALL_LINGUAS="cs de en_GB es hr ja ko ms no pl pt pt_BR sr sr@Latn sv"
+ALL_LINGUAS="cs de en_GB es hr ja ko ms nb pl pt pt_BR sr sr@Latn sv"
 AC_SUBST(ALL_LINGUAS)
 
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, AC_PACKAGE_NAME, [foo])
