diff -aurN gnome-volume-manager-1.5.15.orig/gnome-volume-manager.schemas.in gnome-volume-manager-1.5.15/gnome-volume-manager.schemas.in
--- gnome-volume-manager-1.5.15.orig/gnome-volume-manager.schemas.in	2006-03-10 17:59:17.000000000 +0100
+++ gnome-volume-manager-1.5.15/gnome-volume-manager.schemas.in	2006-05-21 12:09:53.387120008 +0200
@@ -76,7 +76,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoplay_cda_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoplay_cda_command</applyto>
 	<type>string</type>
-	<default>totem %d</default>
+	<default>sound-juicer -d %d</default>
 	<locale name="C">
 	  <short>CD play command</short>
 	  <long>Command to run when an audio CD is inserted.</long>
@@ -133,7 +133,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoplay_dvd_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoplay_dvd_command</applyto>
 	<type>string</type>
-	<default>totem %d</default>
+	<default>totem --play %m</default>
 	<locale name="C">
 	  <short>DVD play command</short>
 	  <long>Command to run when a video DVD is inserted.</long>
@@ -204,7 +204,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoipod_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoipod_command</applyto>
 	<type>string</type>
-	<default></default>
+	<default>rhythmbox</default>
 	<locale name="C">
 	  <short>iPod sync command</short>
 	  <long>Command to run when an iPod is connected.</long>
@@ -250,7 +250,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoscanner_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoscanner_command</applyto>
 	<type>string</type>
-	<default></default>
+	<default>xsane</default>
 	<locale name="C">
 	  <short>scanner command</short>
 	  <long>Command to run when a scanner is connected.</long>
