diff -aurN gnome-volume-manager-1.2.1.orig/src/manager.c gnome-volume-manager-1.2.1/src/manager.c
--- gnome-volume-manager-1.2.1.orig/src/manager.c	2005-03-30 23:20:25.000000000 +0200
+++ gnome-volume-manager-1.2.1/src/manager.c	2005-04-04 19:45:12.606208240 +0200
@@ -38,7 +38,6 @@
 # define N_(String) (String)
 #endif
 
-#define GVM_DEBUG
 #ifdef GVM_DEBUG
 # define dbg(fmt,arg...) fprintf(stderr, "%s/%d: " fmt,__FILE__,__LINE__,##arg)
 #else
