--- src/manager.c.orig	2004-12-10 17:03:43.642189560 +0100
+++ src/manager.c	2004-12-10 17:03:57.150136040 +0100
@@ -460,13 +460,14 @@
 static gboolean
 gvm_device_mount (char *device)
 {
-	char *argv[3];
+	char *argv[4];
 	GError *error = NULL;
 	gint exit_status;
 
 	argv[0] = BIN_MOUNT;
-	argv[1] = device;
-	argv[2] = NULL;
+	argv[1] = "-s";
+	argv[2] = device;
+	argv[3] = NULL;
 
 	if (!g_spawn_sync (g_get_home_dir (), argv, NULL, 0, NULL,
 			   NULL, NULL, NULL, &exit_status, &error)) {
