diff -urN gnome-volume-manager-0.9.8.orig/configure.in gnome-volume-manager-0.9.8/configure.in
--- gnome-volume-manager-0.9.8.orig/configure.in	2004-08-13 22:45:15.475033312 +0200
+++ gnome-volume-manager-0.9.8/configure.in	2004-08-13 22:46:17.661579520 +0200
@@ -61,7 +61,7 @@
 fi
 changequote([,])dnl
 
-ALL_LINGUAS="ar az bg bn ca cs da de en_CA en_GB es eu fi fr gu hr hu is it ja ko ms nl no pa pl pt pt_BR sq sr sr@Latn sv"
+ALL_LINGUAS="ar az bg bn ca cs da de en_CA en_GB es eu fi fr gu hr hu is it ja ko ms nb nl pa pl pt pt_BR sq sr sr@Latn sv"
 AC_SUBST(ALL_LINGUAS)
 
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, AC_PACKAGE_NAME, [foo])
