diff -urN gnome-volume-manager-1.5.1.orig/gnome-volume-manager.schemas.in gnome-volume-manager-1.5.1/gnome-volume-manager.schemas.in
--- gnome-volume-manager-1.5.1.orig/gnome-volume-manager.schemas.in	2005-08-31 13:15:26.947238160 +0200
+++ gnome-volume-manager-1.5.1/gnome-volume-manager.schemas.in	2005-08-31 13:15:53.579189488 +0200
@@ -353,7 +353,7 @@
         <key>/schemas/desktop/gnome/volume_manager/eject_command</key>
 	<applyto>/desktop/gnome/volume_manager/eject_command</applyto>
 	<type>string</type>
-	<default>/bin/eject %d</default>
+	<default>/usr/bin/eject %d</default>
 	<locale name="C">
 	  <short>Eject command</short>
 	  <long>Command to run when the eject button on a optical drive is
