diff -urN gnome-volume-manager-2.22.0/gnome-volume-manager.schemas.in gnome-volume-manager-2.22.0.new/gnome-volume-manager.schemas.in
--- gnome-volume-manager-2.22.0/gnome-volume-manager.schemas.in	2007-11-02 02:55:30.000000000 +0100
+++ gnome-volume-manager-2.22.0.new/gnome-volume-manager.schemas.in	2007-12-27 01:52:11.000000000 +0100
@@ -109,7 +109,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoplay_cda_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoplay_cda_command</applyto>
 	<type>string</type>
-	<default>gnome-cd --unique --play --device=%d</default>
+	<default>sound-juicer -d %d</default>
 	<locale name="C">
 	  <short>CD play command</short>
 	  <long>Command to run when an audio CD is inserted.</long>
@@ -166,7 +166,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoplay_dvd_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoplay_dvd_command</applyto>
 	<type>string</type>
-	<default>totem %d</default>
+	<default>totem --play %m</default>
 	<locale name="C">
 	  <short>DVD play command</short>
 	  <long>Command to run when a video DVD is inserted.</long>
@@ -237,7 +237,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoipod_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoipod_command</applyto>
 	<type>string</type>
-	<default></default>
+	<default>rhythmbox</default>
 	<locale name="C">
 	  <short>iPod sync command</short>
 	  <long>Command to run when an iPod is connected.</long>
@@ -283,7 +283,7 @@
         <key>/schemas/desktop/gnome/volume_manager/autoscanner_command</key>
 	<applyto>/desktop/gnome/volume_manager/autoscanner_command</applyto>
 	<type>string</type>
-	<default></default>
+	<default>xsane</default>
 	<locale name="C">
 	  <short>scanner command</short>
 	  <long>Command to run when a scanner is connected.</long>
